docker/build_and_test.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node@sha256:2ab3d9a1bac67c9b4202b774664adaa94d2f1e426d8d28e07bf8979df61c8694
+FROM node:12.18.4@sha256:59fa78a2149e3470ba7346fb17938e2c48e17096006083003ee1673cc172d676
```

Comment:
This needs to be `FROM node:12.18.4-alpine3.12@sha256:59fa78a2149e3470ba7346fb17938e2c48e17096006083003ee1673cc172d676` (previous upgrade accidentally switched from Alpine to non-Alpine).

You’ll also need to change `python` to `python2`.

Reply:
Done